Run/navstart.ps1

```diff
@@ -22,6 +22,13 @@ $hostname = hostname
 . (Get-MyFilePath "HelperFunctions.ps1")
 . (Get-MyFilePath "SetupVariables.ps1")
 
+if (-not (Test-Path $myPath -PathType Container) -and ($myFilesPackage -ne "")) {
```

Comment:
if their alread is a "my"-folder I'd prefer to not overwrite it but that obviously could easily be changed